tests/lib/rules/func-call-spacing.js

```diff
@@ -223,6 +223,27 @@ ruleTester.run("func-call-spacing", rule, {
     invalid: [
 
         // default ("never")
+        {
+            code: "f     /**/ ();",
```

Comment:
Can we add some tests with newlines in addition to these tests with spaces? Might be nice to throw in some line comments, as well!

Reply:
@kaicataldo I guess this rule doesn't apply fixes for newline [demo](https://eslint.org/demo#eyJ0ZXh0IjoiLyplc2xpbnQgZnVuYy1jYWxsLXNwYWNpbmc6IFtcImVycm9yXCIsIFwibmV2ZXJcIl0qL1xuXG5mbiAvKmFzZGEqcy9cbigpOyIsIm9wdGlvbnMiOnsicGFyc2VyT3B0aW9ucyI6eyJlY21hVmVyc2lvbiI6MTEsInNvdXJjZVR5cGUiOiJzY3JpcHQiLCJlY21hRmVhdHVyZXMiOnt9fSwicnVsZXMiOnsiaW5kZW50IjoyfSwiZW52Ijp7fX19)
So this PR wont change the behavior of the newlines.
I did try adding tests for newline, but in that cases, the input is same as output. that was throwing linting error of internal rules of same `code` as `output`. 

Also, I guess having no `output` will throw error by [this rule](https://github.com/not-an-aardvark/eslint-plugin-eslint-plugin/blob/master/docs/rules/consistent-output.md) ?

Reply:
It should be `output: null` in a test case where there was no autofix made.

This rule generally doesn't remove newlines because it can hide bugs, but there are still some cases where it does remove them (see #13194).

Reply:
> It should be `output: null` in a test case where there was no autofix made.

cool, I will add them with `null`,

> This rule generally doesn't remove newlines because it can hide bugs, but there are still some cases where it does remove them (see #13194).

I agree it should not fix with a newline.